index.html

```diff
@@ -40,7 +47,7 @@ <h1>I'm Naan. <br>Glad to see <br>you!</h1>
 
     <!--Works Section-->
     <div class="works">
-        <div class="card_works">
+        <div class="card_works work_1">
             <div class="snapshot_portfolio"></div>
             <div class="left_block underline">
                 <div class="project_title">Tonic</div>
```

Comment:
- [ ] as requested by the previous reviewer consider double-checking again on the below-requested change and if something is not clear remember to tag me and I will be happy to make it more clear and this will help you to implement the given design 👇 
![asa](https://user-images.githubusercontent.com/60146030/221380304-b831f179-4c17-427f-9af0-794107a6f1d1.PNG)

Reply:
This is an optional requirement as the previous reviewer said. Why are failing me on an optional requirement?? It can be seen on the screenshot as it says optional. @divinecharlotte